site2/website/sidebars.json

```diff
@@ -368,11 +368,7 @@
       "items": [
         "reference-terminology",
         "reference-cli-tools",
-        {
-          "type": "link",
-          "href": "https://pulsar.apache.org/reference",
-          "label": "Pulsar configuration"
-        },
+        "reference-configuration",
```

Comment:
After #18101, I think both "reference-cli-tools" and this new "reference-configuration" page redirect to http://pulsar.apache.org/reference?

But we can improve the docsify side later...